app/scripts/metamask-controller.test.js

```diff
@@ -33,22 +33,32 @@ const firstTimeState = {
 const ganacheServer = new Ganache();
 
 const threeBoxSpies = {
+  _registerUpdates: sinon.spy(),
```

Comment:
This mock really isn't ideal, but I needed to add all methods referenced in the background API to it to prevent it from blowing up. Hopefully we can replace it altogether in the Jest migration.